.editorconfig

```diff
@@ -47,8 +47,8 @@ dotnet_separate_import_directive_groups = true
 # Suggest more modern language features when available
 dotnet_style_object_initializer = true:suggestion
 dotnet_style_collection_initializer = true:suggestion
-dotnet_style_coalesce_expression = true:suggestion
-dotnet_style_null_propagation = true:suggestion
+dotnet_style_coalesce_expression = true:error
```

Comment:
Why are these and other suggestions being changed and what relationship do they have with https://github.com/dotnet/project-system/issues/4933?

Reply:
@davkean the relationship to #4933 was edited in by me as I created that issue today and later noticed @tmeschter created this PR, which coincidentally fixes it.

Reply:
Okay so my question is; I don't see the relationship between the commits/comments and these changes. Why are we now erroring on the lack usage of null propagation?

Reply:
The .ruleset files set the corresponding IDE diagnostics (IDE0029, IDE0030, etc.) to be errors which would have overridden the "suggestion" specified in the .editorconfig. I saw no point in setting the severity twice within the same file via different mechanism, so where possible I've updated the existing "nice" name rather than adding another `dotnet_diagnostics.FOO1234.severity = master".

Reply:
Fair enough.